provisioning/roles/firefox/tasks/main.yml

```diff
@@ -0,0 +1,38 @@
+- name: Install Firefox dependencies
+  apt: name={{ item }} state=present
```

Comment:
I'm not sure about this but couldn't you directly use `{{ firefox_dependencies }}` here?